server/graphql/v2/input/ExpenseReferenceInput.ts

```diff
@@ -0,0 +1,32 @@
+import { GraphQLInt, GraphQLString, GraphQLInputObjectType } from 'graphql';
+import { idDecode, IDENTIFIER_TYPES } from '../identifiers';
+
+const ExpenseReferenceInput = new GraphQLInputObjectType({
```

Comment:
I've tried to follow the pattern defined in https://github.com/opencollective/opencollective/issues/2935 for this one. Let me know if `Reference` is not the final name.